site/content/entry/yuzu-progress-report-may-2020/index.md

```diff
@@ -0,0 +1,178 @@
+*More after the commercial break.*
+<!--more-->
+
+## The worst kept secret
+
+It has its own [article](https://yuzu-emu.org/entry/yuzu-prometheus/), and it has been guessed to hell and back before the official announcement.
+`Project Prometheus` is a proper multithreaded emulation of the 4 CPU cores the Nintendo Switch offers.
+This brings a massive performance boost to users with CPUs with 4 physical cores or more, but for this to happen, a lot of groundwork was needed.
+Besides changes previously discussed in past reports, old external libraries (which yuzu needs to operate) needed to be updated,
+and with that, some changes were needed for our Linux users. 
+
+Thanks to [jroweboy’s](https://github.com/jroweboy) work, yuzu now uses [Conan](https://conan.io/),
+helping the project [manage dependencies](https://github.com/yuzu-emu/yuzu/pull/3735), and letting Linux distributions use their native ones when possible.
+
+Thanks to the previous VMM rewrite reducing memory use, the dependencies updated, and all the groundwork done, [Blinkhawk](https://github.com/FernandoS27) 
+pressed the metaphorical nuclear launch button and [released Project Prometheus](https://github.com/yuzu-emu/yuzu/pull/3955).
+yuzu use up to 6 to 7 CPU threads in ideal conditions 
+compared to the previous 2 to 3. Expect a performance boost in a lot of games, while others will stay mostly the same due to 
+being coded to only use a single thread of the possible 3 the Switch offers to games.
+
+Now, some clarifications are needed for this change. Multicore support can’t be merged into our [Mainline](https://github.com/yuzu-emu/yuzu-mainline) release for now due to incompatibilities between Multicore and the [Master](https://github.com/yuzu-emu/yuzu) branch of yuzu. Work is being done to resolve the conflicts, but please have patience.
+Additionally, users with 2 cores, and either 2 or 4 threads, should not enable multicore as it will most likely result in a performance 
+loss for them due to the lack of physical cores on their CPUs. Our [hardware recommendations](https://yuzu-emu.org/help/quickstart/#hardware) 
+have been updated accordingly.
+
+## Unreal Engine fixes
+
+[Rodrigo](https://github.com/ReinUsesLisp) implemented [rendering more than one slice of 3D textures](https://github.com/yuzu-emu/yuzu/pull/4027), fixing the most glaring 
+issue in Unreal Engine 4 games, known as “the Rainbow”. This also improves the previous implementation that was used in 
+Xenoblade games.
+
+{{< single-title-imgs
+    "Your Excellency (OCTOPATH TRAVELER)"
+    "./01.png"
+    "./02.png"
+  >}}
+
+## Animal Crossing: New Horizons changes
+
+[Rodrigo](https://github.com/ReinUsesLisp) fixed `Animal Crossing: New Horizons` terrain borders in Vulkan by implementing 
+constant attributes. This is not a native extension, [constant attributes](https://github.com/yuzu-emu/yuzu/pull/3930) have to be emulated in Vulkan as there is currently no official support for it.
```

Comment:
You changed the 2nd instance of `constant attributes` instead of the first lol.

The first is the one that comes after `implementing`

Reply:
welp